app/screens/AppNavigator/screens/Settings/screens/ConfirmPinScreen.tsx

```diff
@@ -36,7 +36,7 @@ export function ConfirmPinScreen ({ route }: Props): JSX.Element {
     }
 
     const copy = { words, network, pin }
-    setSpinnerMessage(translate('screens/PinConfirmation', 'Updating pin code...'))
+    setSpinnerMessage(translate('screens/PinConfirmation', 'Reencrypting wallet...'))
```

Comment:
Same here, let's use updating passcode instead of reencrypting as their wallets didn't lose encryption